arches_lingo/src/arches_lingo/components/header/DarkModeToggle.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import ToggleButton from "primevue/togglebutton";
+
+import { DEFAULT_THEME } from "@/arches/themes/default.ts";
+
+const { $gettext } = useGettext();
+
+const isDarkModeEnabled = ref(
+    window.matchMedia &&
+        window.matchMedia("(prefers-color-scheme: dark)").matches,
+);
+
+function toggleDarkMode() {
```

Comment:
instead of classes, how do you feel with the `data-*` attr (eg data-theme)? I've seen that pretty often with other theme switchers

Reply:
I was trying to pair this with the doc'd pattern in primevue, which we already merged to core, see:

https://github.com/archesproject/arches/blob/7289bd3d5b678e6c67f6455f1c10c6df9d5659af/arches/app/src/arches/themes/default.ts#L53

Reply:
oh fair